src/components/TreeNode.tsx

```diff
@@ -231,7 +303,31 @@ const TreeNode = ({
           top: y,
           left: 0,
         }
-      : undefined
+      : isMoveAnimating
+        ? moveType === 'moveThoughtCursor'
+          ? {
+              transformOrigin: 'left',
+              transform: 'scale3d(1.5, 1.5, 1)',
+              transition: `transform {durations.layoutNodeAnimation} ease-out`,
```

Comment:
The `moveThoughtCursor` snaps a bit too quickly at the end of the animation. Maybe it's the choice of easing function, I'm not totally sure. Could you see if you can smooth it out?

Reply:
Much better!